conf/cfme_data.yaml.template

```diff
@@ -170,6 +170,7 @@ management_systems:
             default:
                 credentials: lenovo
                 ipaddress: 10.0.0.1
+                hostname: 10.243.6.103
```

Comment:
QUESTION: Why IP address and not hostname? Or why different IP address?

Reply:
**REQUIRED:** Yeh please set to 10.0.0.1

Reply:
About the difference between `ipaddress` and `hostname`, it was a typo, that I will fix. I've added this new property because I couldn't figure out how to access the `ipaddress` data on the `LenovoProvider` constructor, but I was able to access the `hostname` prop, so I go on and add this new property even it is duplicating data. (see [lenovo.py](https://github.com/ManageIQ/integration_tests/blob/master/cfme/physical/provider/lenovo.py#L34))

Do you know some way to prevent this duplication of data? Maybe a solution to access the `ipaddress` prop into `LenovoProvider`